doc/man3/X509_check_purpose.pod

```diff
@@ -65,6 +65,7 @@ keyUsage, extendedKeyUsage, and basicConstraints.
 X509_PURPOSE_get_count() returns the number of currently defined purposes.
 
 X509_PURPOSE_get_unused_id() returns the smallest purpose id not yet used.
+The I<libctx> parameter should be used to provide the library context.
```

Comment:
Maybe add `It is currently unused as purpose mapping table is currently global.'

Reply:
added